MainConsoleApp/ConsoleApp2/DbUtils.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Linq;
+
+namespace ConsoleApp2
+{
+    public class DbUtils
+    {
+        public static void AddNewCustomerDocking(StarWarsPerson person)
+        {
+            if (person == null) throw new ArgumentNullException(nameof(person));
+
+            using var context = new StarWarsDbContext();
+            person.EntryTime = DateTime.Now;
+            context.Person.Add(person);
+            context.SaveChanges();
+        }
+
+        public static bool IsDocked(StarWarsPerson person)
+        {
+            if (person == null) throw new ArgumentNullException(nameof(person));
+            using var context = new StarWarsDbContext();
+            return context.Person.Any(p => p.Name == person.Name && p.ExitTime == null);
+        }
+
+        public static StarWarsPerson CheckOutCustomer(StarWarsPerson person)
+        {
+            if (person == null) throw new ArgumentNullException(nameof(person));
+            using var context = new StarWarsDbContext();
+            var item = context.Person.FirstOrDefault(p => p.Name == person.Name && p.ExitTime == null);
+            if (item != null)
+            {
+                item.ExitTime = DateTime.Now;
+                context.SaveChanges();
+            }

```

Comment:
På flera ställen har ni, preics som vi gjort en ny "using context...." istället för att skicka med den in i en metod eller ha den i början på klassen och återanvända den. Blir mer dry om man gör så i såg vi när vi såg de andras arbeten!